source/rst/lss_examples.rst

```diff
@@ -97,29 +97,30 @@ as a linear state space model in the following way:
                       I_t \cr} \right]}  
 
 We use this form as we are interested in the paths of :math:`C` and
-:math:`I` as well as :math:`Y`.
+:math:`I` as well as :math:`Y`
 
-We want to ask the following questions of this model: \* Does
-:math:`Y_t` have a stationary distribution? If so, what is it? \* How
-long does it take for :math:`Y_t` to reach its stationary distribution?
+We want to ask the following questions of this model: 
+
+\* Does :math:`Y_t` have a stationary distribution? If so, what is it? 
+\* How long does it take for :math:`Y_t` to reach its stationary distribution?
 \* What is the effect of an investment shock, :math:`\sigma \epsilon_t`,
 on present and future values of :math:`C_t,I_t` and :math:`Y_t`?
 
-The methods of the Linear State Space class will be able to help us out.
+The methods of the ``LinearStateSpace`` class will be able to help us out
 
 To start, lets assume the following parameter values:
 
 .. math::  a = 0.9,b = 0.5, \gamma = 10,\sigma = 0.5,G = 5 
 
-.. code:: ipython3
+.. code-block:: python3
```

Comment:
Sorry I didn't say this earlier, this particular block should be `ipython` because it contains a Jupyter magic (specific to ipython)